src/app/components/Search/Search.jsx

```diff
@@ -19,7 +19,7 @@ class Search extends React.Component {
     this.state = {
       field: this.props.field,
       searchKeywords: this.props.searchKeywords,
-      inputError: this.props.searchError === 'true',
+      inputError: !!this.props.searchError,
```

Comment:
Here you're making it `true`/`false` but it's still being set to `''` or `'noKeyword'` in the component.